### PR TITLE
cluster and host list view implementation

### PR DESCRIPTION
**Pull request by kamleshverma1**
added cluster and host listing view with API integration

### CODE REVIEW

File: src/modules/host/host.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        margin: auto 5px;
+    }
+    .list-view-pf-additional-info-item.deflex {
+        flex-basis: 10%;
+    }
+    .focus {
+        font-weight: 600;
+        font-size: 2rem;
+    }
+}
```
Comment (kmkanagaraj) [inline src/modules/host/host.scss]: cluster.scss and host.scss has common content. Can they be extracted and reused?

Comment (kamleshverma1) [inline src/modules/host/host.scss]: ya i will update the PR .. thanks for your review..

**Comment by kamleshverma1** [comment]
@kmkanagaraj please review

**Comment by kmkanagaraj** [comment]
@kamleshverma1 looks good to me

**Comment by kamleshverma1** [comment]
@kmkanagaraj Can you please approved this PR. so that i can merge it .

**Comment by kamleshverma1** [comment]
@nthomas-redhat Can you please approved this PR . so that i can merge this (got reviewed by kanagaraj)

**Comment by kamleshverma1** [comment]
@anivargi Please review.

**Comment by kamleshverma1** [comment]
@anivargi Renaming from host to node across the project , will not look good in respect to UI.
we can not show name as 'Node' in navigation and also in listing page .
either we can rename node to host across the project.

**Comment by anivargi** [comment]
@kamleshverma1 I don't think renaming the word node across the project is possible, lets make it consistent for now, we can think about handling the UI wording later.

**Comment by kamleshverma1** [comment]
@anivargi reworked on your given comment . please review it again.

**Comment by kamleshverma1** [comment]
@anivargi Please review it again ..

**Comment by kamleshverma1** [comment]
@anivargi Updated this file - test/unit/mock-data.js . you can review it again

**Pull request merged**